packages/http-client-python/generator/pygen/m2r.py

```diff
@@ -1,65 +0,0 @@
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-# --------------------------------------------------------------------------
-"""An MD to RST plugin.
-"""
```

Comment:
Hi @catalinaperalta Since autorest.python which is generator tool for swagger is still using m2r, it will break autorest.python if remove m2r directly.

Could you make a PR to copy m2r.py to atuorest.python repo to make sure autorest.python could work after this PR merged?

Reply:
@iscai-msft for awareness.

Reply:
yes, that would be part of this work as well, and I've already talked to @catalinaperalta about it. thanks for the catch @msyyc !